pkg/content/provider.go

```diff
@@ -42,4 +42,7 @@ type Provider interface {
 	Image(ctx context.Context, ref string) (*ocispec.Descriptor, error)
 	// ContentStore gets the content store object of containerd.
 	ContentStore() content.Store
+	// SetCacheRef gets the cache reference of the source image,
```

Comment:
So we'd better provide a separate `GetCacheRef` ?

Reply:
fixed.Here is GetCacheRef and SetCacheRef two method.